subiquity/common/types.py

```diff
@@ -315,42 +313,28 @@ class Disk:
     vendor: Optional[str] = None
 
 
-@attr.s(auto_attribs=True)
-class GuidedChoice:
-    disk_id: str
-    use_lvm: bool = False
-    password: Optional[str] = attr.ib(default=None, repr=False)
-    use_tpm: bool = False
-
-
-class StorageEncryptionSupport(enum.Enum):
-    DISABLED = 'disabled'
-    AVAILABLE = 'available'
-    UNAVAILABLE = 'unavailable'
-    DEFECTIVE = 'defective'
-
-
-class StorageSafety(enum.Enum):
-    UNSET = 'unset'
-    ENCRYPTED = 'encrypted'
-    PREFER_ENCRYPTED = 'prefer-encrypted'
-    PREFER_UNENCRYPTED = 'prefer-unencrypted'
-
+class GuidedCapability(enum.Enum):
+    DIRECT = enum.auto()
+    LVM = enum.auto()
+    LVM_LUKS = enum.auto()
+    CORE_BOOT_ENCRYPTED = enum.auto()
+    CORE_BOOT_UNENCRYPTED = enum.auto()
+    # These two are not valid as GuidedChoiceV2.capability:
```

Comment:
Are these `prefer` capabilities staying or going, eventually?

Reply:
We talked about this I think? They will probably go eventually and the list of capabilities end up being replaced by something richer that indicates availability and default.